### DISCUSSION TITLE
Wie kann ich diese Warning message loswerden?: Warning message: GET_DEVICE_DESCRIPTIONS failed: NoConnection [SSLError ...

### DISCUSSION THREAD

**Discussion by Wombater** [SukramJ/aiohomematic#1591] [Q&A] [answered]
Hallo, 
vielleicht kann mir ja jemand einen Tipp geben wie ich die Warnung loswerden kann. 

Ich habe die Raspberrymatic als ADD-On in Homeassistant auf einem Mini-PC (Beelink) laufen. Funktioniert soweit alles prima mit der „Homematic(IP) Local“-Integration (102 Geräte und 1485 Entitäten“). Geräte funktionieren soweit ich sehen kann einwandfrei.
Auf einem externen Raspi läuft ein CCU-Historian und kann auch zugreifen auf die Raspimatic, die im Add-on läuft.

Im Log der Homeassistant Software im Homeassistant core log erscheint immer wieder die folgende Warnung:

Logger: hahomematic.client
Quelle: custom_components/homematicip_local/control_unit.py:749
Integration: Homematic(IP) Local (Dokumentation, Probleme)
Erstmals aufgetreten: 14:13:08 (4 Vorkommnisse)
Zuletzt protokolliert: 20:13:08

**GET_DEVICE_DESCRIPTIONS failed: NoConnection [SSLError on RaspberryMatic-HmIP-RF: (8, 'EOF occurred in violation of protocol (_ssl.c:2406)')]**

So sieht das im vollständigen Protokoll aus: 
2024-06-10 16:04:37.134 WARNING (MainThread) [hahomematic.client] GET_DEVICE_DESCRIPTIONS failed: NoConnection [SSLError on RaspberryMatic-HmIP-RF: (8, 'EOF occurred in violation of protocol (_ssl.c:2406)')]
2024-06-10 18:04:37.143 WARNING (MainThread) [hahomematic.client] GET_DEVICE_DESCRIPTIONS failed: NoConnection [SSLError on RaspberryMatic-HmIP-RF: (8, 'EOF occurred in violation of protocol (_ssl.c:2406)')]

Der Fehler wird exakt alle 2 Stunden protokolliert. Ich vermute, dass beim Lesen der Gerätebeschreibungen plötzlich ein Ende des Datenstromes auftritt (EOF). 

Ich habe das erst mal hier als Diskussion gepostet, da es ja nur eine Warnung ist und es ja auch an einem Konfigurationsfehler in HA oder an der Raspberrymatic-Konfiguratonsfehler liegen kann. 

Wie gesagt, vielleicht hat ja jemand eine Idee wo ich suchen könnte.

**Comment by SukramJ** [comment, marked as answer]
TLS deaktivieren

**Reply by Wombater** [reply]
Das hat funktioniert. Vielen Dank für die superschnelle Hilfe. Die Integration funktioniert wirklich sehr gut!

Mit diesen Einstellungen in der Integration und Raspberrymatic Systemsteuerung/Sicherheit kommen keine Warnungen mehr im Log:
![Einstellungen – Home Assistant](https://github.com/danielperna84/hahomematic/assets/150610704/3bf7b794-52e0-46b2-8e8c-e023a083a8f8)
![Raspberrymatic HTTPS Umleitungoff jpeg](https://github.com/danielperna84/hahomematic/assets/150610704/e428d35b-8733-473c-8bec-636c38cca1ed)
